public/js/actions/management.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+import $ from 'jquery';
+import * as actionTypes from 'constants/action-types';
```

Comment:
I find it odd that you can't do `import 'constants/action-types' as actionTypes` but I looked it up and I don't think you can